scripts/jenkins/jenkinsfiles/Jenkinsfile-Nightly

```diff
@@ -39,7 +39,7 @@ try {
             if (pipelineContext == null) {
               def initPipelineContext = load('h2o-3/scripts/jenkins/groovy/initPipelineContext.groovy')
               pipelineContext = initPipelineContext(scmEnv, 'MODE_NIGHTLY', true)
-              // Execute nightly for master at 22:XX, 0:XX, 2:XX, 4:XX and 6:XX
+              // Execute nightly for master at 22:XX
```

Comment:
only change to comment, not the actual schedule

Reply:
muhaha :) thanks @honzasterba